pkg/data/templates/forSarah.yaml

```diff
@@ -0,0 +1,66 @@
+components:
```

Comment:
can we rename this file to what its representing

Reply:
it was accidentally committed. I'll delete it.